android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMKeyboard.java

```diff
@@ -507,60 +507,37 @@ public boolean setKeyboard(String packageID, String keyboardID, String languageI
         KMManager.getLatestKeyboardFileVersion(getContext(), packageID, keyboardID) : null;
     }
 
-    String kbKey = KMString.format("%s_%s", languageID, keyboardID);
+    if(kOskFont == null || kOskFont.isEmpty())
+      kOskFont = kFont;
 
-    setKeyboardRoot(packageID);
```

Comment:
`setKeyboardRoot(packageID)` needs to remain before the font path munging. Otherwise, issue #5418 results.